src/content/en/tools/lighthouse/index.md

```diff
@@ -85,6 +89,11 @@ command line tool. The Chrome Extension lets you audit logged-in pages and
 provides a more user-friendly interface for reading reports. The command line
 tool enables you to integrate Lighthouse into continuous integration systems.
 
+<div class="yt-container">
```

Comment:
Please use the standardized DevSite youtube widget - https://developers.google.com/web/resources/widgets#videos